tests/cli/info/owners/test_code.py

```diff
@@ -0,0 +1,186 @@
+        "testfile.txt": ["@owner1", "@owner2"],
+    }
+    expected_result = ownership_data
+    _test_owner_template(
+        dda,
+        temp_dir=temp_dir,
+        ownership_data=ownership_data,
+        expected_result=expected_result,
+    )
+
+
+def test_wildcard_ownership(dda, temp_dir):
+    ownership_data = {
+        "*.txt": ["@owner1"],
+        "testfile.txt": ["@owner2"],
+    }
+    expected_result = {
+        "testfile.txt": ["@owner2"],
+        "otherfile.txt": ["@owner1"],
+    }
+    _test_owner_template(
+        dda,
+        temp_dir=temp_dir,
+        ownership_data=ownership_data,
+        expected_result=expected_result,
+    )
+
+
+def test_directory_ownership(dda, temp_dir):
+    ownership_data = {
+        "subdir1": ["@owner1"],
+        "subdir2": ["@owner2"],
+        "subdir2/testfile2.txt": ["@owner3"],
+    }
+    expected_result = {
+        "subdir1": ["@owner1"],
+        "subdir1/testfile1.txt": ["@owner1"],
+        "subdir2": ["@owner2"],
+        "subdir2/testfile2.txt": ["@owner3"],
+    }
+    _test_owner_template(
+        dda,
+        temp_dir=temp_dir,
+        ownership_data=ownership_data,
+        expected_result=expected_result,
+    )
+
+
+def test_ownership_location(dda, temp_dir):
+    ownership_data = {
+        "testfile.txt": ["@owner1"],
+    }
+    expected_result = ownership_data
+    _test_owner_template(
+        dda,
+        temp_dir=temp_dir,
+        extra_command_parts=["--config", "custom/CODEOWNERS"],
+        codeowners_location=Path("custom/CODEOWNERS"),
+        ownership_data=ownership_data,
+        expected_result=expected_result,
+    )
+
+
+def test_complicated_situation(dda, temp_dir):
+    ownership_data = {
+        "*": ["@DataDog/team-everything"],
+        "*.md": ["@DataDog/team-devops", "@DataDog/team-doc"],
+        ".gitlab": ["@DataDog/team-devops"],
+        ".gitlab/security.yml": ["@DataDog/team-security"],
+    }
+    expected_result = {
+        "test.txt": ["@DataDog/team-everything"],
+        "README.md": [
+            "@DataDog/team-devops",
+            "@DataDog/team-doc",
+        ],
+        ".gitlab": ["@DataDog/team-devops"],
+        ".gitlab/security.yml": ["@DataDog/team-security"],
+        ".gitlab/ci.yml": ["@DataDog/team-devops"],
+    }
+    _test_owner_template(
+        dda,
+        temp_dir=temp_dir,
+        ownership_data=ownership_data,
+        expected_result=expected_result,
+    )
+
+
+def test_human_output(dda, temp_dir):
+    ownership_data = {
+        "testfile.txt": ["@owner1"],
+        "subdir/testfile2.txt": ["@owner2"],
+        "subdir/anotherfile.txt": ["@owner1", "@owner3"],
+    }
+
+    _create_codeowners_file(ownership_data, temp_dir / DEFAULT_CODEOWNERS_LOCATION)
+    _create_temp_items(ownership_data.keys(), temp_dir)
+    with temp_dir.as_cwd():
+        result = dda("info", "owners", "code", *ownership_data.keys())
+    assert result.exit_code == 0, result.stdout
+    assert (
+        result.stdout
```

Comment:
Let's use the utility like other tests:

https://github.com/DataDog/datadog-agent-dev/blob/f8026d5d3566f06f719a317d64b04c2c370e11b2/tests/cli/config/test_set.py#L26-L31